Makefile

```diff
@@ -240,6 +240,11 @@ dm: dm-master dm-worker dmctl dm-portal dm-syncer
 dm-master:
 	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/dm-master ./dm/cmd/dm-master
 
+dm-master-with-webui:
```

Comment:
Shall we always embed webui into dm-master

Reply:
This is also an option, but introduces the problem that every time  build DM code you need to download the node dependencies and then `node build` (this is too slow and need adjust CI config)

so I feel like I should only do this when building a release package?  I see that `pd` does the same thing https://github.com/tikv/pd/blob/master/Makefile#L56